src/leiningen/ezbake_core.clj

```diff
@@ -43,12 +60,18 @@
   TODO: Add configuration option to clone a git repository containing ezbake
   resources instead of pulling them from the lein-ezbake jar."
   [project]
-  (let [template-type (get-in project [:lein-project
```

Comment:
I'm curious about this change.  It seems likely to break existing flows.

Reply:
It looks like we never use it.  Given this is a public project, I think it would be safer to not change it, and only use the new path for the new entry.

Reply:
? :lein-project doesn't exist as a keyword in project.clj, this code was just incorrect, returning `nil` always

Reply:
Existing code should be fine unless some other type is specified in project.clj, which shouldn't be the case considering there is only one, this still defaults to :jar if no value

Reply:
I did a search on projects that use this, and looked for `lein-project` all of them seem to use `lein-ezbake` as the keys int he project map.